scripts/validate

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+cd $(dirname $0)
+
+# Thanks to https://gist.github.com/rverst/1f0b97da3cbeb7d93f4986df6e8e5695
+function chsv_check_version() {
+  if [[ $1 =~ ^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(-((0|[1-9][0-9]*|[0-9]*[a-zA-Z-][0-9a-zA-Z-]*)(\.(0|[1-9][0-9]*|[0-9]*[a-zA-Z-][0-9a-zA-Z-]*))*))?(\+([0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*))?$ ]]; then
+    echo "$1"
+  else
+    echo ""
+  fi
+}
+
+echo "Checking if pkg directory contains only semver compatible version"
+
+for DOCKER_VERSION in $(ls -1 ../pkg/); do
+    SEMVER=$(chsv_check_version $DOCKER_VERSION)
+    if [ -z $SEMVER ]; then
+        echo "[ERR] Version found in pkg directory is not semver (${DOCKER_VERSION})"
+        exit 1
+    fi
+done
+
+echo "Checking basic shell validation"
+find ../dist -name \*.sh -exec sh -n {} \;
```

Comment:
May be would be better to add `check_semversion`  as bash function and could be called for both paths, `check_semversion pkg/` and `check_semversion dist/`

```
function check_semversion() {
    if [ -z $1 ]; then
        echo "[ERR] Checking sameversion: directory to check is empty"
        exit 1
    fi
    if [ ! -d $1 ]; then
        echo "[ERR] Checking sameversion in $1 directory: directory doesn't exist"
        exit 1
    fi
    for DOCKER_VERSION in $(ls -1 $1); do
        SEMVER=$(chsv_check_version $DOCKER_VERSION)
        if [ -z $SEMVER ]; then
            echo "[ERR] Version found in $1 directory is not semver (${DOCKER_VERSION})"
            exit 1
        fi
    done
}
```